src/app/connections/list/list.component.html

```diff
@@ -5,9 +5,9 @@
         <div class="card-pf card-pf-view card-pf-view-xs">
           <div class="card-pf-body">
 
-            <!-- Card Dropdown -->
-
+            <!-- Card Heading -->
             <div class="card-pf-heading-kebab">
+              <!-- Card Actions --
```

Comment:
Ah totally missed that, funny really only noticed it because github highlighted it so obviously.  Yeah, may as well comment out the block in a separate comment.